Plugin: 语音功能

**插件名称：**

语音功能

**插件功能：**

语音功能适配插件，包括语音事件响应器，语音识别、语音合成等功能

**PyPI 项目名：**

nonebot-plugin-record

**插件 import 包名：**

nonebot_plugin_record

**插件项目仓库/主页链接：**

https://github.com/itsevin/nonebot_plugin_record

**标签：**

[{"label":"t:语音","color":"#fff35d"},{"label":"语音识别","color":"#37c0f6"},{"label":"语音事件响应器","color":"#18e13c"}]

# 📃 商店发布检查结果

> Plugin: 语音功能

**⚠️ 在发布检查过程中，我们发现以下问题：**
<pre><code><li>⚠️ 插件加载测试未通过。<details><summary>测试输出</summary>插件 nonebot-plugin-record 的信息如下：
    name         : nonebot-plugin-record                        
     version      : 1.0.0                                        
     description  : 基于 Nonebot2 的语音功能适配插件，包括语音事件响应器，语音识别、语音合成等功能 
    
    dependencies
     - httpx &gt;=0.23.3,&lt;0.24.0
     - nonebot-adapter-onebot &gt;=2.0.0-beta.1,&lt;3.0.0
     - nonebot2 &gt;=2.0.0-beta.1,&lt;3.0.0
     - pilk &gt;=0.2.3,&lt;0.3.0
插件 nonebot_plugin_record 加载出错：
01-15 14:55:47 [SUCCESS] nonebot | NoneBot is initializing...
01-15 14:55:47 [INFO] nonebot | Current Env: prod
01-15 14:55:48 [ERROR] nonebot | Failed to import &quot;nonebot_plugin_record&quot;
Traceback (most recent call last):
  File &quot;/home/runner/work/nonebot2/nonebot2/plugin_test/runner.py&quot;, line 4, in &lt;module&gt;
    valid = load_plugin(&quot;nonebot_plugin_record&quot;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
    return manager.load_plugin(module_path)
&gt; File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 141, in load_plugin
    module = importlib.import_module(name)
  File &quot;/usr/lib/python3.10/importlib/__init__.py&quot;, line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1050, in _gcd_import
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1027, in _find_and_load
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1006, in _find_and_load_unlocked
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 688, in _load_unlocked
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 233, in exec_module
    super().exec_module(module)
  File &quot;&lt;frozen importlib._bootstrap_external&gt;&quot;, line 883, in exec_module
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 241, in _call_with_frames_removed
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_record/__init__.py&quot;, line 20, in &lt;module&gt;
    from .config import plugin_config
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_record/config.py&quot;, line 19, in &lt;module&gt;
    assert(
AssertionError: 请填写完整配置！</details></li></code></pre>
<details><summary>详情</summary><pre><code><li>✅ 标签: t:语音-#fff35d, 语音识别-#37c0f6, 语音事件响应器-#18e13c。</li><li>✅ 项目 <a href="https://github.com/itsevin/nonebot_plugin_record">主页</a> 返回状态码 200。</li><li>✅ 包 <a href="https://pypi.org/project/nonebot-plugin-record/">nonebot-plugin-record</a> 已发布至 PyPI。</li></code></pre></details>

---

💡 如需修改信息，请直接修改 issue，机器人会自动更新检查结果。
💡 当插件加载测试失败时，请发布新版本后在当前页面下评论任意内容以触发测试。

💪 Powered by [NoneBot2 Publish Bot](https://github.com/nonebot/nonebot2-publish-bot)
<!-- PUBLISH_BOT -->

--- a/website/static/plugins.json
+++ b/website/static/plugins.json
@@ -4088,5 +4088,28 @@
       }
     ],
     "is_official": false
+  },
+  {
+    "module_name": "nonebot_plugin_record",
+    "project_link": "nonebot-plugin-record",
+    "name": "语音功能",
+    "desc": "语音功能适配插件，包括语音事件响应器，语音识别、语音合成等功能",
+    "author": "itsevin",
+    "homepage": "https://github.com/itsevin/nonebot_plugin_record",
+    "tags": [
+      {
+        "label": "t:语音",
+        "color": "#fff35d"
+      },
+      {
+        "label": "语音识别",
+        "color": "#37c0f6"
+      },
+      {
+        "label": "语音事件响应器",
+        "color": "#18e13c"
+      }
+    ],
+    "is_official": false
   }
 ]

